packages/components/psammead-story-promo-list/testHelpers/fixtureData.js

```diff
@@ -0,0 +1,63 @@
+const storyPromoData = [
```

Comment:
Can you replace this data with something more representative of stories please :) Realistic length headlines and summaries make it easier to judge the read world look on an index page